### PR TITLE
update zope.component and zope.interface

### PR DESCRIPTION
**Pull request by jensens**
First of a step by step trial how far we may update our ZTK stack in the current setup.

Also experimental usage of plone/plone.versioncheck#25

### CODE REVIEW

File: versions.cfg [hunk 1]
```diff
@@ -325,3 +327,11 @@ plone.app.drafts                      = 1.1.1
 plone.app.tiles                       = 2.2.1
 plone.tiles                           = 1.8.0
 plone.app.ldap                        = 1.4.0
+
+[versionannotations]
+# keep this alphabetical please
```
Comment (mauritsvanrees) [inline versions.cfg:329]: Keeping it alphabetical seems very good. Can you start by moving zope.component above zope.interface? :-)
Someone can also do that manually before merging, if the pull request is good.

Comment (jensens) [reply versions.cfg:329]: lol, thanks. ok, need to follow my own rules. this is just experimental.

**Comment by jensens** [comment]
trapped into https://community.plone.org/t/problems-with-mocktestcase-in-plone-dexterity/1484

fixed in https://github.com/plone/plone.mocktestcase/commit/97dbd21bd1116b76f82d843e663d1bd7fd93d6fc

File: versions.cfg (hunk 1 above)
Comment (jensens) [reply versions.cfg:329]: done

**Comment by gforcada** [comment]
Isn't that part of the PLIP https://github.com/plone/Products.CMFPlone/issues/1351 ? I mean, is there any little benefit on updating just two out of 200 zope dependencies at a time? Or is this just a testing pull request for your enhancements on plone.versioncheck?

**Comment by jensens** [comment]
It makes it easier to have code we use in pyramid with latest of those packages and plone to work together. Also it does not update to latest, but to latest possible in our current environment.

no, not related to plone.versioncheck

**Comment by pbauer** [comment]
The jenkins job of https://github.com/plone/Products.CMFPlone/issues/1351 (http://jenkins.plone.org/view/PLIPs/job/plip-zope4/) has been failing for a while now. I think it should be more of a priority for us to keep these tests green now that Zope sees more development. Before the conference-sprint I will not have time to work on that.

**Comment by gforcada** [comment]
@pbauer due to that activity (on zope packages) the jenkins job is failing. Basically lots of imports are failing so not even tests can run :-/

It no longer makes sense to target master branches of most packages probably...

**Pull request merged**